tests/jest/BookReader.keyboard.test.js

```diff
@@ -0,0 +1,190 @@
+import BookReader from '@/src/BookReader.js';
+import * as utils from '@/src/BookReader/utils.js';
+
+let br;
+beforeAll(() => {
+  document.body.innerHTML = '<div id="BookReader">';
+  br = new BookReader();
+});
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+/**
+ * Only run init() once. Otherwise multiple EventListeners will be added.
+ */
+test('Initialzation enables IntersectionObserver and defaults', () => {
+  const observe = jest.fn();
+  window.IntersectionObserver = jest.fn(() => ({
```

Comment:
Why do we need this? Shouldn't the jest setup file be doing this?

Reply:
I could not find a clear way to do it.
The jest setup file is to allow all other tests to run without crashing, although not needed now what we're wrapping it so I am removing it.